### DISCUSSION TITLE
Installed v2.1.6  getting File ID is not found.

### DISCUSSION THREAD

**Discussion by jovett** [hyunbinseo/zoom-rec-dl#17] [Q&A] [answered]
2023-07-01T14:07:12.616Z Found 1 valid URLs.

2023-07-01T14:07:12.620Z ┌ zi9b58Lzlws0XVJtsEbY
2023-07-01T14:07:13.027Z └ File ID is not found.

I can paste the url in a  browser and it opens and plays . the UTL does contain password 

Woking on a MAC with Node v18.16.1

sample link pw blanked out
https://us02web.zoom.us/rec/share/zi9b58Lzlws0XVJtsEbYvqTmpeZm8Gfu593pLjWJ9M9Ej8VimlNIo6JWGPm5thkq.myHY2klARNXlBcn8?pwd=xxxxxxxx

TIA

**Comment by hyunbinseo** [comment]
Can you check if the following setting is disabled?

> Set recording as on-demand by default - Users must register before they can watch the recording

The above recording URL requires. a Zoom login, not a passcode entry.

**Comment by jovett** [comment, marked as answer]
OK I have checked all the settings you require and they are correct

 Just got it to work.

Add this to your documentation
When getting the shared recording link needs to be set to:
*Who can view:* "Everyone with the recording link"

Question?  can you build a routine to download all the links in an
account using the api.
Happy to pay for development.  Let me know.

You can this API for getting the download URL of the recordings:
https://developers.zoom.us/docs/meeting-sdk/apis/#operation/recordingGet

[image: Screenshot 2023-07-02 at 8.22.29 AM.png]


correct it has been disabled.

My Setting you have in your set up
[image: Screenshot 2023-07-02 at 8.15.45 AM.png]
[image: Screenshot 2023-07-02 at 8.18.31 AM.png]

**Reply by hyunbinseo** [reply]
Thanks for the information. I am planning to update the README as following. Could you check if it is correct? I cannot see the images you have attatched.

---

Reference the [documentation](https://support.zoom.us/hc/en-us/articles/11692220055821) and update the following settings in the Zoom web portal.

```
ADMIN / Account Management / Account Settings
e.g. https://us06web.zoom.us/account/setting

[v] Allow cloud recording sharing - By disabling this setting, nobody else except the host can access the shareable link.

[v] Cloud recording downloads - Allow anyone with a link to the cloud recording to download
[ ] └── Only the host can download cloud recordings

[ ] Require users to authenticate before viewing cloud recordings

[ ] Set recording as on-demand by default - Users must register before they can watch the recording

[v] Require passcode to access shared cloud recordings
[v] └── Embed passcode in the shareable link for one-click access
```

---

There are no plans to use the official Zoom API to access the cloud recordings. Our partner provides the Zoom recording URLs for us as of now. The goal of this project was to automate the download process of these URLs.

This can change. To receive updates on this, please subscribe to this issue. https://github.com/hyunbinseo/zoom-rec-dl/issues/18

**Comment by jovett** [comment]
export the zoom recording list as csv and use that list to extract
recording ids in the list and append as a  correct url you need to download
, or just add it the urls.txt file.

**Comment by jovett** [comment]
List all recordings
https://developers.zoom.us/docs/meeting-sdk/apis/#operation/recordingsList
